certification/runtime/config.go

```diff
@@ -56,6 +56,28 @@ func NewConfigFrom(vcfg viper.Viper) (*Config, error) {
 	return &cfg, nil
 }
 
+func NewManualContainerConfig(image, responseFormat, artifactsDir string, submit, writeJUnit bool) *Config {
+	return &Config{
+		Image:          image,
+		Submit:         submit,
+		WriteJUnit:     writeJUnit,
+		ResponseFormat: responseFormat,
+		Artifacts:      artifactsDir,
+	}
+}
+
+func NewManualOperatorConfig(image, responseFormat, artifactsDir string, writeJUnit bool) *Config {
+	return &Config{
+		Image:          image,
+		Submit:         false, // operator results are not submitted
+		WriteJUnit:     writeJUnit,
+		ResponseFormat: responseFormat,
+		Artifacts:      artifactsDir,
+		Bundle:         false,
+		Scratch:        false,
+	}
+}
+
```

Comment:
@sebrandon1 let's go ahead and remove these for now. I believe this is part of what's driving down coverage.

Reply:
Yup sounds good.  They won't be needed in the future implementation.